<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>961</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>960</width>
       <height>540</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_enable_camera">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>730</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_processing_time">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>620</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>--/--</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>620</y>
       <width>120</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Processing time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>620</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_fps">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>620</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>--/--</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_video_status">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>710</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_enable_other_features">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>670</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Other features</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_video">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>620</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Select_video</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_path_video">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>620</y>
       <width>371</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>620</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Path to video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_graph">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>650</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Select_graph</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>650</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Detector</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_path_graph">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>650</y>
       <width>371</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_path_gender_model">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>680</y>
       <width>371</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_gender_model">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>680</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Select_model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>680</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="midLineWidth">
      <number>-4</number>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_path_age_model">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>710</y>
       <width>371</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_load_age_model">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>710</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Select_model</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>710</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Age&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_enable_record">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>650</y>
       <width>96</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cur_frame">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>580</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_video">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>580</y>
       <width>881</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_enable_detection_s3fd">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>690</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Face detector S3FD</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_enable_detection">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>700</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Face detector</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_enable_playing">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>720</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Video</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
